lib/elasticache_redis.py

```diff
@@ -0,0 +1,599 @@
+"""
+    https://aws.amazon.com/blogs/database/monitoring-best-practices-with-amazon-elasticache-for-redis-using-amazon-cloudwatch/
+"""
+
+from typing import List
+from grafanalib.core import (
+    Alert,
+    AlertCondition,
+    Dashboard,
+    LowerThan,
+    Graph,
+    GreaterThan,
+    MILLISECONDS_FORMAT,
+    OP_OR,
+    PERCENT_FORMAT,
+    Repeat,
+    Row,
```

Comment:
Can we please have cpu utilization and cpu credit as 2 different graphs?

![Screen Shot 2021-04-22 at 9 19 24 AM](https://user-images.githubusercontent.com/5619473/115672426-d7016680-a34b-11eb-899c-2fe4e1db67cf.png)